app/styles/mixins/_custom-font-face.scss

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Licence?

Reply:
removed, strange thing - it is just scss mixin